src/Game.js

```diff
@@ -0,0 +1,31 @@
+function Game (scorer) {
+  if (typeof (require) !== 'undefined') {
+    Scorer = require('./Scorer')
+  }
+
+  this._scorer = scorer || new Scorer()
+}
+
+Game.prototype = {
+  constructor: Game,
+
+  roll: function (roll) {
+    this._scorer.addRoll(roll)
```

Comment:
.addRoll is from the Score object - circular dependancy. From the workshop I think each object is not supposed to depend on other objects as much as possible (I think this was actually your question).